src/main/resources/assets/sodium/lang/ru_ru.json

```
@@ -0,0 +1,66 @@
+  "sodium.options.brightness.tooltip": "Управляет яркостью (гаммой) игры.",
+  "sodium.options.gui_scale.tooltip": "Устанавливает коэффициент масштабирования для интерфейса. При выборе \"авто\" всегда будет использоваться наибольший размер.",
+  "sodium.options.fullscreen.tooltip": "Если включено, игра будет открыта во весь экран (если доступно).",
+  "sodium.options.v_sync.tooltip": "Если включена, частота кадров будет синхронизирована с частотой обновления монитора, обеспечивая более плавную игру за счет небольшой задержки ввода. Может понизить производительность слабых систем.",
+  "sodium.options.fps_limit.tooltip": "Ограничивает максимальное количество кадров в секунду. Это поможет уменьшить энергопотребление и нагрузку на систему в обеспечении многозадачности. При включённой вертикальной синхронизации этот параметр будет игнорироваться, если он выше частоты обновления вашего экрана.",
+  "sodium.options.view_bobbing.tooltip": "Если включено, обзор игрока будет покачиваться при движении. Отключение этого параметра может помочь людям, страдающим от укачивания во время игры.",
+  "sodium.options.attack_indicator.tooltip": "Управляет расположением индикатора атаки на экране.",
+  "sodium.options.autosave_indicator.tooltip": "Если включено, при сохранении игрой мира на диск будет показываться индикатор.",
+  "sodium.options.graphics_quality.tooltip": "Качество графики по умолчанию управляет некоторыми наследующими настройками и необходимо для совместимости с модами. Опции ниже со значением \"По умолчанию\" будут использовать этот параметр.",
+  "sodium.options.clouds_quality.tooltip": "Управляет качеством отрисовки облаков в небе.",
+  "sodium.options.weather_quality.tooltip": "Управляет качеством эффектов дождя и снега.",
+  "sodium.options.leaves_quality.name": "Качество листьев",
+  "sodium.options.leaves_quality.tooltip": "Управляет качеством листьев.",
+  "sodium.options.particle_quality.name": "Качество частиц",
+  "sodium.options.particle_quality.tooltip": "Управляет количеством частиц, которые могут присутствовать на экране в любой момент времени.",
+  "sodium.options.smooth_lighting.tooltip": "Управляет мягким освещением и затенением блоков. Это немного увеличивает время сборки чанка, однако не влияет на частоту кадров.",
+  "sodium.options.biome_blend.value": "%s блок(ов)",
+  "sodium.options.biome_blend.tooltip": "Управляет диапазоном анализа биомов для расцветки блока. Более высокие значения сильно увеличивают затрачиваемое на сборку чанков время ради незначительного улучшения качества.",
+  "sodium.options.entity_distance.tooltip": "Управляет дальностью видимости сущностей. Высокие значения увеличивают расстояние отображения за счёт частоты кадров.",
+  "sodium.options.entity_shadows.tooltip": "Если включено, простые тени будут отображаться под мобами и другими сущностями.",
```

The same words and meaning, just a better formulation
```suggestion
  "sodium.options.entity_shadows.tooltip": "Если включено, под мобами и другими сущностями будут отображаться простые тени.",
```